workflow/cron/operator.go

```diff
@@ -113,9 +113,27 @@ func (woc *cronWfOperationCtx) persistUpdate() {
 		woc.log.WithError(err).Error("failed to marshall cron workflow status data")
 		return
 	}
-	err = wait.ExponentialBackoff(retry.DefaultBackoff, func() (bool, error) {
+
+	woc.persist(data)
+}
+
+func (woc *cronWfOperationCtx) persistUpdateActiveWorkflows() {
+	data, err := json.Marshal(map[string]interface{}{"status": map[string]interface{}{"active": woc.cronWf.Status.Active}})
+	if err != nil {
+		woc.log.WithError(err).Error("failed to marshall cron workflow status.active data")
+		return
+	}
+
+	woc.persist(data)
+}
+
+func (woc *cronWfOperationCtx) persist(data []byte) {
+	err := wait.ExponentialBackoff(retry.DefaultBackoff, func() (bool, error) {
 		cronWf, err := woc.cronWfIf.Patch(woc.cronWf.Name, types.MergePatchType, data)
 		if err != nil {
+			if errors.IsTimeout(err) {
+				return false, nil
+			}
```

Comment:
If this is not added, the `ExponentialBackoff` will never actually try again since both existing return points return either `true` or an `err`, which will prevent the backoff from retrying